app/templates/components/search-result.hbs

```diff
@@ -0,0 +1,21 @@
+<div>
+    <h4><a>{{result.title}}</a></h4>
+    {{#each result.contributors as |contributor|}}
+        <a>{{contributor.familyName}}, {{contributor.givenName}}</a>
+    {{/each}}
+    <p {{action "toggleShowDescription"}} style="cursor: pointer; cursor: hand;">{{displayDescription}}</p>
```

Comment:
Avoid clickable text elements where possible.

Use markup with semantic meaning: `button` for something that performs an action, and `a` for something that takes you to a new place.

That will also save you from the need to embed styles in markup (custom styles should be in the CSS file). If you use a button, the bootstrap css class `btn-link` may be useful.